lib/gds_api/performance_platform/data_out.rb

```diff
@@ -43,6 +43,119 @@ class DataOut < GdsApi::Base
       def service_feedback(transaction_page_slug)
         get_json("#{endpoint}/data/#{transaction_page_slug}/customer-satisfaction")
       end
+
+      # Fetches data from the performance platform for a given page slug
+      #
+      # Makes a +GET+ request.
+      #
+      # @param slug [String] The slug for which page is needed.
+      # @param group_by [String] The group by condition by which
+      # to group the results
+      # @param collect [String] The operation that should be applied
+      # on the results after they've been grouped
+      #
+      # # @example
+      #
+      #  performance_platform_data_out.search_terms('/european-health-insurance-card', 'pagePath', 'uniquePageviews:sum')
+
+      def page_statistics(slug, group_by, collect, is_multipart)
+        params = {
+            group_by: group_by,
+            collect: collect,
+            duration: 42,
+            period: "day",
+            end_at: Date.new.to_time.getutc
+        }
+
+        filter_param = is_multipart ? :filter_by_prefix : :filter_by
+        params[filter_param] = "pagePath:" + slug
+
+        get_json("#{endpoint}/data/govuk_info/page-statistics?#{query_string(params)}")
+      end
+
+      # Fetches data from the performance platform for a given page slug
+      #
+      # Makes a +GET+ request.
+      #
+      # @param slug [String] The slug for which page is needed.
+      # @param group_by [String] The group by condition by which
+      # to group the results
+      # @param collect [String] The operation that should be applied
+      # on the results after they've been grouped
+      #
+      # # @example
+      #
+      #  performance_platform_data_out.searches('/european-health-insurance-card', 'pagePath', 'searchUniques:sum')
+
+      def searches(slug, group_by, collect, is_multipart)
+        params = {
+            group_by: group_by,
+            collect: collect,
+            duration: 42,
+            period: "day",
+            end_at: Date.new.to_time.getutc
+        }
+
+        filter_param = is_multipart ? :filter_by_prefix : :filter_by
+        params[filter_param] = "pagePath:" + slug
+
+        get_json("#{endpoint}/data/govuk_info/searchTerms?#{query_string(params)}")
+      end
+
+      # Fetches data from the performance platform for a given page slug
+      #
+      # Makes a +GET+ request.
+      #
+      # @param slug [String] The slug for which page is needed.
+      # @param group_by [String] The group by condition by which
+      # to group the results
+      # @param collect [String] The operation that should be applied
+      # on the results after they've been grouped
+      #
+      # # @example
+      #
+      #  performance_platform_data_out.search_terms('/european-health-insurance-card', 'searchKeyword', 'searchUniques:sum')
+
+      def search_terms(slug, group_by, collect)
+        params = {
+            filter_by: "pagePath:" + slug,
```

Comment:
It seems odd to me that this doesn't take the is_multipart argument to switch `filter_by` into `filter_by_prefix`.  It is [exactly what metadata API's go implementation does](https://github.com/alphagov/metadata-api/blob/87cc4db4f6801ac8936419a0a697e09e9012f405/performance_platform/statistics.go#L116-L123) though so we should leave it for now.  That said I think we should take a note to check if we should be toggling `filter_by` / `filter_by_prefix` for this API call and we can make that change later.  For now we want things to be a drop in replacement, we can improve it later.